annotation/src/main/java/org/cbioportal/genome_nexus/annotation/service/internal/VEPVariantAnnotationService.java

```diff
@@ -74,9 +75,17 @@ public VEPVariantAnnotationService(EnrichmentService enrichmentService,
 
     public VariantAnnotation getAnnotation(String variant)
     {
-        VariantAnnotation variantAnnotation = variantAnnotationRepository.findOne(variant);
+        boolean storeVariantJson = true;
```

Comment:
maybe name this variable `saveAnnotationJson` for consistency with the method name?